dali/python/nvidia/dali/plugin/mxnet.py

```diff
@@ -22,14 +22,22 @@
 import logging
 import numpy as np
 
+
+##################################################
+##################################################
+################## Common utils ##################
+##################################################
+##################################################
+
+
 # MXNet currently does not expose WaitToWrite C API call
 # in Python API
 def _wait_to_write(arr):
     if not isinstance(arr, mx.nd.NDArray):
         raise RuntimeError("Can only wait for NDArray")
     mx.base._LIB.MXNDArrayWaitToWrite(arr.handle)
 
-def feed_ndarray(dali_tensor, arr):
+def _feed_ndarray(dali_tensor, arr):
```

Comment:
Someone may be using that. In ptyorch DALI Coco iterator someone does.

Reply:
Reverting it back to "public"